### PR TITLE
Remove config and references to kubernetes-incubator

### PR DESCRIPTION
**Pull request by cblecker**
ref https://github.com/kubernetes/community/issues/1922

### CODE REVIEW

**Comment by cblecker** [comment]
/hold cancel
Test are passing, and this can be merged as soon as it's approved.

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm
🎉

**Pull request merged**
